### DISCUSSION TITLE
Move externalicon i18n hint to plugin

### DISCUSSION THREAD

**Discussion by Mister-Hope** [vuepress/core#622] [Q&A] [answered]
I am not getting the point that the default theme is overiding the icon provided by `external-link` plugin, just providing an extra i18n a11y hint.
(Also all the theme are expected doing the same way if they wanna provide a full support of a11y

We can just make a refactor to let the plugin accept a `locale` option and users can set i18n hint text for its default `<ExternalIcon />`, and the default theme manually provide a option to let users set it then pass to the plugin, it should be much more graceful than current way.

If you think the above is fine, I will open a PR.

**Comment by meteorlxy** [comment]
The `@vuepress/plugin-external-link-icon` was extracted from `@vuepress/client` recently, because I think it's strange to maintain such feature in core.

Lots of things could be refined continuously. If a11y of external link icon is a common requirements, it's better to wrap it inside the plugin itself for sure.

**Comment by Mister-Hope** [comment, marked as answer]
Solved and released
